tools/build/ci_install_libpq.sh

```diff
@@ -18,9 +18,16 @@ case "$libpq" in
         ;;
 
     oldest)
+        curl -sL https://www.postgresql.org/media/keys/ACCC4CF8.asc \
+            | gpg --dearmor \
+            | sudo tee /etc/apt/trusted.gpg.d/apt.postgresql.org.gpg > /dev/null
+
+        rel=$(lsb_release -c -s)
+        echo "deb http://apt.postgresql.org/pub/repos/apt ${rel}-pgdg main 10" \
+            | sudo tee -a /etc/apt/sources.list.d/pgdg.list > /dev/null
+        sudo apt-get -qq update
```

Comment:
This block of line, from `curl` to `apt-get update`, can be made common between the oldest and the newest branch.

Reply:
Not quite, because the "echo" line is different. But I've factor out common bits still. Also simplified key retrieval a bit.